packages/edit-site/src/components/layout/index.js

```diff
@@ -113,6 +115,8 @@ export default function Layout() {
 			: 'site-editor';
 	useCommandContext( commandContext );
 
+	const [ backgroundColor ] = useGlobalStyle( 'color.background' );
```

Comment:
This only works properly after the "global styles" load no? In other words, you'll get two different results during the loading phase which is not desirable no?

Reply:
Should we also have a second call to retrieve the gradient for the themes that use gradients and not background colors.

Reply:
> This only works properly after the "global styles" load no? In other words, you'll get two different results during the loading phase which is not desirable no?

Yes, that's a good point. However, we currently display a separate instance of the canvas spinner, one that appears specifically when global styles haven't been loaded yet. I think now that we have the spinner appearing inside the canvas, we can remove this other one. I'll suggest that in a separate PR since it feels like a separate discussion IMHO.

Reply:
> Should we also have a second call to retrieve the gradient for the themes that use gradients and not background colors.

Done in https://github.com/WordPress/gutenberg/pull/51709/commits/6d13173f4ef891776cd31107b95f21753c07c2ed